Server/model/game.py

```diff
@@ -197,7 +197,8 @@ def get(cls):
             cls._instance = {
                 # All
                 0:  [
-                        option(0, "Paved Holes", 0)
+                        option(0, "Paved Holes", 0),
+                        option(49, "Starting Room", 0)
```

Comment:
Starting room need not, and probably shouldnt been in this list

Reply:
We dont want to be able to vote in a second starting room, so it shouldnt be in here